Create mock data and add database seed command (#83)
ohtutraininghub/traininghub: docs/CONTRIBUTING.md

1. `npm run dev:install`

> > **_Note_:** Runs script located at `scripts/dev_install.sh`. Your database will be seeded with some mock course and tag records.

Reviewer: ```suggestion
> **_Note_:** Runs script located at `scripts/dev_install.sh`. Your database will be seeded with mock data.
```

More generally that there will be mock data as in the future there will be more

### Start and stop project locally

1. `npm run dev`
2. To stop use CTRL + C in terminal

> > **_Note_:** Runs script located at `scripts/dev_start.sh` > **_Note_:** To access database directly see [Prisma studio](#prisma)

Reviewer: ```suggestion
> **_Note_:** Runs script located at `scripts/dev_start.sh`  
> **_Note_:** To access database directly see [Prisma studio](#prisma)
```
